### DISCUSSION TITLE
Should “Financial Value” be a quality?

### DISCUSSION THREAD

**Discussion by JamesJEgan** [CommonCoreOntology/CommonCoreOntologies#353] [Announcements]
Should “Financial Value” be a quality?

Financial value doesn’t seem to fit the definition of quality since it does not inhere in an independent continuant that could be exchanged in a market, is not internally grounded, and is not fully realized in that entity. It doesn’t seem that any independent continuant could have an inherent financial value. 

Financial value is externally grounded based on social relations and conventions. It is determined by factors like market supply and demand, labor and production costs, as well as other qualities of the entity that contribute to its demand. So it seems some further process is always needed to determine financial value. 

Here are some specific examples: 
- The exact same house could have a certain value at one time and a different value at another time due entirely to external shifts in the housing market. 
- A diamond existing in a time or place where there is no market for diamonds has no financial value at all. 

To the extent that an entity’s financial value is based on qualities realized in an entity, those specific qualities could just be accounted for. For example, the hardness of a diamond makes it more in demand as drill bit material than wood, which is too soft. It doesn’t seem to make sense to talk about the diamond having some “economic exchangeability-ness” quality. 

Definitions: 

CCO: Financial Value "A quality that inheres in an independent continuant to the degree that that independent continuant can serve as a medium of exchange in an economic system at a particular time.” 

BFO: Quality "a quality is a specifically dependent continuant that, in contrast to roles and dispositions, does not require any further process in order to be realized.”

Quality 
“A specifically dependent continuant that, if it inheres in an entity at all, is fully exhibited, manifested, or realized in that entity. In order for a quality to exist, one or more independent continuants must also exist. Examples include the mass of a kidney, the color of this portion of blood, and the shape of a hand.” (Robert Arp;Barry Smith;Andrew D. Spear. “Building Ontologies with Basic Formal Ontology.” 529)

**Comment by cameronmore** [comment]
@JamesJEgan 
This is a great observation. I have modified the definition and made a PR to the develop branch with this change. My proposed definition is only a slight modification from the original:

`Financial Value = A role that inheres in an independent continuant and is realized in processes during which its bearer serves as a medium of exchange in an economic system at a particular time.`

**Comment by astudor** [comment]
If Financial Value is exhibited by some digital item at some time t, isn't that a generically dependent continuant?

**Comment by cameronmore** [comment]
This is a good question, what kind of digital item do you have in mind? Something like money stored in a digital record of a bank account? Or more like an NFT or an online book for rent?

@astudor

**Comment by astudor** [comment]
First I thought of btc / cryptocurrencies, then why not NFTs or whatever digital representation that agents want to use as means of exchange, and score it with a _financial value_.  

Also.. would it be better to replace _economic system_ with something like system of exchange that uses a common denominator?

**Comment by cameronmore** [comment]
It's worth asking in each of these cases what is actually being exchanged? Regarding NFTs, I can download them as many times as I want--I can store lots of Information Bearing Entities on multiple computers, but these aren't in themselves economic acts. Both NFTs and cryptocurrencies depend on very complicated and highly monitored information systems and computer processes to determind who own what, who transfers what to whom, and so on... I am not certain whether the ICE of the NFT or the actual bits of the NFT bear that Financial Value Role.

This is worth thinking more about.

Regarding expanding the scope to 'system of exchange', I think that financial value is a specific kind of vale, and economic exchange is a specific kind of exchange--both are sub-types of more generic classes. I would rather introduce those generic classes rather than expand these.

**Comment by astudor** [comment]
I don't think the object of exchange or its counterpart have any relevance. I see it this way:

There are two conditions for an object **(physical or digital)** to gain the role of Financial Value:
1. at least one agent has right of possession/usage over it and at least one other agent wants to gain that right - thus, the relation between the agents and the object, make the object display **value**.
2. to gain a **financial** value, the object must be quoted in a common denominator that both agents are knowledgeable about. 

At this point, the object discussed by agents has financial value and:
- there is no need for a transaction to take place
- there is a need for an exchange system with a common denominator, but no need for an entire economic system.  
- the transaction can still be of type barter, or other forms of deal, not necessarily the initial quoted number of shells/sats/dollars.
- the definition seems compliant with future generic classes like 'exchange system', 'economic system', 'finance'.

Any guidance to what I am missing is highly appreciated, as I am quite new to ontologies overall. Thank you.

**Comment by cameronmore** [comment]
These questions merit a discussion about the role and its realization conditions.

I think I agree with (1), but I'm not sure I know exactly what this would mean for an ICE, rather than an IBE or physical artifact. If a person is paying for possession or control of an Information Bearing Entity, like a CD, then the physical CD bears the Financial Value role which is realized when it changes hands. If a person is paying for access to an ICE, they seem to be paying for access through its bearer (since ICEs are always carried by some bearer). On the other hand, we don't exactly value the bearer highly, we value the information that is carried highly (I don't just buy inked up paper at the bookstore, I'm buying a book that carries an appealing novel that I want to read). What do you (or others) think?

> there is no need for a transaction to take place

The current definition is not 100% clear regarding whether a transaction must take place for the role to be realized, thought to me, you're right, it seems like exchange **or** appraisal must occur, and this is something to think about. To get a loan, I may need to put up some highly valued asset as collateral. At no time does it leave my possession (though by putting it up, I may promise to forfeit its possession if I do not meet my end of a promise). Or, during a property evaluation, I don't lose possession of the land but it is appraised and the taxes paid on that land are adjusted accordingly.

> there is a need for an exchange system with a common denominator, but no need for an entire economic system.

I mean merely that there must be _some_ economic exchange system, not one or or the entire economic system. For example, some nations do not legally allow certain currencies to be validly exchanged for practical and/or political reasons, though black and grey markets exist everywhere (like this example in 2019 in [Zimbabwe](https://www.bbc.com/news/world-africa-48757080). Though, one could argue that since if all currency is eventually convertible into other kinds of currency, there is just one system... Financial value is realized in _some_ economic exchange system, but not every or all. This leaves open the question of how to define economic exchange system itself.

Here are just some thoughts to get started:

Is an economic system just a "system of exchange that uses a common denominator"? I'm not sure, financial systems distinguish themselves through the use of money, rather than any denominator. There are at least three types of systems that intersect here that we should disentangle: an exchange system, an economic system, and a financial system (not to mention the difficulty of what a system is). Just thinking out loud, exchange systems are those whose participants may gain and lose rights and possessions over objects; an economic system is an exchange system which involves the exchange, consumption, production, distribution (and so on) of resources, not just exchange; and a financial system is a part of the economic system in which some money is used as a common denominator of comparison and exchange.

**Comment by astudor** [comment]
>  a discussion about the role and its realization conditions

When you say this, you mean it is the case to open a discussion thread elsewhere?
I believe the role is a deontic role, as defined in  https://ceur-ws.org/Vol-2518/paper-SOLEE7.pdf: 
> Deontic role =def. role that inheres in an agent and which is externally grounded in the normative expectations that other agents within a social context have concerning how that agent should behave. 

I would argue the realization of Financial Value comes with first thought of supposed bidder/asker regarding how much to price the object for, in amount of a referenced monetary units. 

> I'm not sure I know exactly what this would mean for an ICE

I don't see any interference. Indeed, an ICE might be the real value bearer of the Financial Value. It is sometimes sold with a bearer in the sense of medium of support, other times can be denoted in a verbal gentleman's agreement.

Financial Value for direct ICE transfer: 
Giovanni sits on a bench at Colosseum, in Rome. He often talks to tourists passing by, acting as a passionate local guide. Proving himself valuable in just 5 minutes of conversation, for $10 he offers to tell them where to find two secret places they can't afford to miss during their short stay. 

Financial Value for tacit/non-defined ICE: 
When getting home in the evening, Giovanni gets on whatsapp a "Thank you" note, and a photo with the tourists he advised earlier. He logs into digital photo marketplace, registered in Kiribati and sells the pic "Happy Family in Rome's Dungeons" for $100. The Kiribati website does not distinguish between commercial and noncommercial use, so the buyer can use the photo as he pleases.

### Can we talk about a functional financial system without the rest of the economic system? 
John has two kids who love dinosaurs, so he created the monetary unit "DINO" in his family. For a good deed the kids "get" a DINO, in John's notebook. The kids sell their toy-cars to each other against Dinos, offer to help in exchange for Dinos, and give back Dinos to dad, in exchange for TV time, or carousel rides in amusement park or even real money for ice-cream.

**Comment by jonathanvajda** [comment]
There was a pull-request that adopts the change to be a subclass of bfo:Role. That is certainly a better parent class than bfo:Quality for this.

However, I think we're jumping the gun to make the change. It is worth moving this issue tracker to discussion. Because we've got a lot of outstanding issues to solve on a theoretical level.

One proposal there was that all financial value is tied to an economic exchange (or asking, bidding, accepting etc.) or an economic system. That theory has some pedigree, but I hesitate that it isn't a consensus.

Two other outstanding issues:
- What can have a financial value? Information entities? Processes? Neither of those can bear roles. Will we bite the bullet?
- What about the intuition that the value is itself _information_ (not a role), since it is the output of an act of appraisal (subclass of act of measuring)?

We might be doing violence to common intuitions, so we need to have an tight argument if it is going to be in CCO.

**Comment by cameronmore** [comment]
I think this conversation should be moved to the discussion section until a proposal is agreed upon.
